deeplearning4j-core/src/test/java/org/deeplearning4j/nn/layers/custom/testclasses/CustomLayer.java

```diff
@@ -79,4 +80,9 @@ public void setNIn(InputType inputType, boolean override) {
     public InputPreProcessor getPreProcessorForInputType(InputType inputType) {
         return null;
     }
+
+    @Override
+    public LayerMemoryReport getMemoryReport(InputType inputType) {
+        throw new UnsupportedOperationException();
+    }
```

Comment:
This gives me idea, that we should document memory requirements for users with custom layers...